core/lib/dal/src/blocks_dal.rs

```diff
@@ -441,7 +441,7 @@ impl BlocksDal<'_, '_> {
     ) -> DalResult<()> {
         match aggregation_type {
             AggregatedActionType::Commit => {
-                let instrumentation = Instrumented::new("set_eth_tx_id#commit")
+                let i = Instrumented::new("set_eth_tx_id#commit")
```

Comment:
Bikeshedding: I don't really like one-letter var naming here; `i` is usually used for index vars. How about `instr` if `instrumentation` is too long?